lntest/mock/walletcontroller.go

```diff
@@ -280,6 +280,14 @@ func (w *WalletController) Stop() error {
 	return nil
 }
 
+// ReadySignal currently signals that the wallet is ready instantly.
+func (w *WalletController) ReadySignal() chan error {
+	readyChan := make(chan error, 1)
+	readyChan <- nil
+
+	return readyChan
```

Comment:
cant you just return nil?

Reply:
same below

Reply:
I think maybe I should motivate that the reason why this returns a `chan error` and not just `error`, is that we want to make sure that a shutdown signal is listened to when waiting for the remote signer to connect in `lnd.go`. Therefore this needs to return a channel, and therefore we can't just return `nil` here.
ReadySignal

Reply:
Hmmm I'm not fully sure if I'm still misunderstanding here: But if this function just returned `nil`, any code that listens to the channel returned by this definition of the `ReadySignal` function would deadlock? Let me know if I'm missing something :)